src/components/schema/_schema_item.scss

```diff
@@ -0,0 +1,79 @@
+/*!
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+.ouiSchemaItem {
+  display: flex;
+  // sass-lint:disable-block no-misspelled-properties
+  gap: $ouiSizeS;
+  align-items: center;
+  padding: $ouiSizeS;
+  border-radius: $ouiBorderRadius;
+
+  &:hover {
+    @include ouiFocusBackground($ouiTextColor);
+
+    .ouiSchemaItem__actions .ouiButtonIcon {
+      opacity: 1;
+      visibility: visible;
+    }
+  }
+}
+
+.ouiSchemaItem__icon {
+  align-self: flex-start;
+  margin: $ouiSize / 2 0; // To align with buttonIcon
+}
+
+.ouiSchemaItem__label {
+  flex: 1;
+  margin: $ouiSize / 2 0; // To align with buttonIcon
+}
+
+.ouiSchemaItem__actions {
+  // sass-lint:disable-block no-misspelled-properties
+  gap: $ouiSizeXS;
+  display: flex;
+
+  .ouiButtonIcon {
+    opacity: 0;
+    transition: opacity $ouiAnimSpeedNormal $ouiAnimSlightResistance;
+
+    &:focus {
+      opacity: 1;
+    }
+  }
+}
+
+.ouiSchemaItem--withPanel {
+  @include ouiBottomShadowSmall;
+  border: $ouiBorderThin;
+
+  &:hover {
+    background-color: tintOrShade($ouiColorLightestShade, 50%, 20%);
+  }
```

Comment:
The naming of this seems a little fishy. I would expect the component prop to just wrap itself with https://oui.opensearch.org/1.0/#/layout/panel. This is just adding border and a hover background color. Is it intended to actually leverage/mimic existing panel styles or is it just a different styling for this component?

Reply:
It is, but its style is different from a normal panel. Based on @KrooshalUX's mock in the parent issue, the colors here come from other similar components and they do it this way. We can probably reuse them, but given that OUI components are mostly atomic anyways, i dont think its a big deal to redefine them unless we are certain that this is actually a composed view of other atomic components.